signimage/Makefile

```diff
@@ -0,0 +1,34 @@
+IMAGE_TAG_BASE ?= quay.io/chrisp262/kmod-signer
```

Comment:
Do we plan to refer to your repo?

Reply:
I don't think we need new Makefile targets. [`cloudbuild.yaml`](https://github.com/kubernetes-sigs/kernel-module-management/blob/main/cloudbuild.yaml), where we will be adding instructions for this new image, already doesn't use them.

Reply:
The Makefile has been merged into the main Makefile so marking as resolved